src/qt/recentrequeststablemodel.cpp

```diff
@@ -214,10 +213,10 @@ void RecentRequestsTableModel::updateDisplayUnit()
     updateAmountColumnTitle();
 }
 
-bool RecentRequestEntryLessThan::operator()(RecentRequestEntry &left, RecentRequestEntry &right) const
+bool RecentRequestEntryLessThan::operator()(const RecentRequestEntry& left, const RecentRequestEntry& right) const
```

Comment:
are these changes required for the centos build to pass?

Reply:
Please look #17634.

Reply:
Ah got it. I'm still new to how bitcoin is developed but I think it would be useful if this PR didn't try to merge to master but instead tried to merge into #17634 so that the changes from the PR don't appear here (but that's just a stylistic nit).

Reply:
From OP:
> Please note that this PR is based on the bugfix #17634.

When/if #17634 is merged this PR will be rebased and commits from #17634 will not appear here.